pbjson-test/build.rs

```diff
@@ -23,6 +23,7 @@ fn main() -> Result<()> {
         .extern_path(".google.protobuf", "::pbjson_types")
         .extern_path(".test.external", "crate")
         .bytes(&[".test"])
+        .protoc_arg("--experimental_allow_proto3_optional")
```

Comment:
This is needed because the version of prost in the CI image is old enough for this to not be the default behaviour